server/views/rivers-and-sea.html

```diff
@@ -47,41 +47,53 @@ <h1 class="govuk-heading-xl govuk-!-margin-top-6">
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
       <h2 class="govuk-heading-l govuk-!-padding-top-6 govuk-!-margin-bottom-4">How likely a river or sea flood is</h2>
-      <p class="govuk-body govuk-!-padding-top-4">The yearly chance of rivers and sea flooding is:</p>
-      <p class="govuk-body">
-        <strong class="govuk-tag govuk-tag--{{ riversSeaRiskStyle }} label-mod-depth">
-          <span>{{ riverAndSeaRisk }}</span>
-        </strong>
-        {% if riverAndSeaRisk == riverAndSeaRiskCC %}
-        staying at
-        {% else %}
-        increasing to
-        {% endif %}
-        <strong class="govuk-tag govuk-tag--{{ riversSeaRiskCCStyle }} label-mod-depth">
-          <span>{{ riverAndSeaRiskCC }}</span>
-        </strong>
-        between 2036 to 2069
-      </p>
+      {% if riverAndSeaRiskCC == "Unavailable" %}
```

Comment:
If it still works it might be better to change these to exact equals ('===') in order to avoid potential bugs in the future. Same with the other ones below.